notebooks/DrizzlePac/Initialization/Initialization.ipynb

```diff
@@ -0,0 +1,319 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DrizzlePac Initialization\n",
+    "\n",
+    "<br>This Jupyter notebook discusses the steps necessary to set up your computing environment to use DrizzlePac software. This is the first step before using any of the other DrizzlePac tutorials. The code cells in this notebook can be used to partially confirm that your environment is properly configured before proceeding to the other tutorials.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "DrizzlePac is a Python software package developed at STScI that is designed to align and combine HST images and is a successor to the older MultiDrizzle software, bringing an array of new features an algorithmic improvements. Since July 2012, all drizzled data products obtained from MAST are produced with AstroDrizzle. \n",
```

Comment:
```suggestion
     "DrizzlePac is a Python software package developed at STScI that is designed to align and combine HST images and is a successor to the older MultiDrizzle software. Since July 2012, all drizzled data products obtained from MAST are produced with AstroDrizzle. \n",
```
Users coming to this notebook probably don't care/know that much about MultiDrizzle, and if they do this isn't news to them.

(or alternatively the second, "an" -> "and")